server/jetstream_cluster.go

```diff
@@ -1672,7 +1672,7 @@ func (rg *raftGroup) setPreferred() {
 }
 
 // createRaftGroup is called to spin up this raft group if needed.
-func (js *jetStream) createRaftGroup(accName string, rg *raftGroup, storage StorageType) error {
+func (js *jetStream) createRaftGroup(accName, assetName string, rg *raftGroup, storage StorageType) error {
```

Comment:
Why not just use cfg.Name?

Reply:
`cfg.Name` is set from `rg.Name`, which seems to be the raft ID, not the actual asset name.

Reply:
Yes its the store name. But if you are trying to tie these together at this level we would need to be more expressive then assetName I believe.

Reply:
E.g. a consumer would also need which stream it belongs to.

Let's chat about this a bit on a Zoom at some point.